src/astro/files/locations/google/gcs.py

```diff
@@ -6,7 +6,7 @@
 from astro.utils.dependencies import gcs
 
 
-class GcsLocation(BaseFileLocation):
+class GCSLocation(BaseFileLocation):
```

Comment:
In the same way that `Http` is capitalized using Title, I suggest we stick to title capitalization for all classes. Otherwise, we can be consistent and rename all of them, which should be uppercase to uppercase.

Reply:
Done.